src/Microsoft.VisualBasic.Core/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>10.4.0</PackageVersion>
-    <AssemblyVersion>10.0.4.0</AssemblyVersion>
+    <PackageVersion>5.0.0</PackageVersion>
```

Comment:
@cston @ericstj should this stay at 10.4.0?

Reply:
10.5